RIT-MDRC/Catbot
# Feat/pneumatic control

# Feature
* Compressor control on the arduino
* Handshake behavior on the arduino
* Handshake receiving on the raspi
* Valve control on the raspi
* No behavioral code has been implemented for this branch yet

# How to Test
* Test Arduino behavior
  - Upload code through the ide 
  - turn on the main system
  - the pressure should go up to ideal pressure (80psi)
  - open the serial monitor on your computer to double check
  - lower the pressure (could be just by opening the valve or slowly leaking the air)
  - compressor should turn back on when it is lower than ideal
* Test Raspi behavior
  - TBD

# Package requirements
* python 3.12 (latest python package)
* gpiozero
